# This is the dynamixel_sdk config file for joint of s500_uam
joint_1:
  ID: 0
  Return_Delay_Time: 0
  Operating_Mode: 3

joint_2:
  ID: 1
  Return_Delay_Time: 0
  Operating_Mode: 0

joint_3:
  ID: 2
  Return_Delay_Time: 0
  Operating_Mode: 3
